vendor/go.etcd.io/bbolt/tx.go

```diff
@@ -185,6 +186,10 @@ func (tx *Tx) Commit() error {
 
 	// If the high water mark has moved up then attempt to grow the database.
 	if tx.meta.pgid > opgid {
+		_ = errors.New("")
+		// gofail: var lackOfDiskSpace string
+		// tx.rollback()
+		// return errors.New(lackOfDiskSpace)
```

Comment:
The `_ = errors.New("")` confused me, but looks like it's to make sure the `errors` import is there, and I guess the `gofail:` comment is used by some tooling they use to un-comment this code in tests; https://github.com/etcd-io/bbolt/commit/79bcca0e518503f3de4c126e7f3a2fa140bf2ba8